packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -253,6 +265,45 @@ void WebView::Touch(int type, int button, double x, double y, double dx,
   eina_list_free(points);
 }
 
+void WebView::SendMouseEvent(int type, int button, double x, double y,
+                             double dx, double dy) {
+  Ewk_Mouse_Button_Type button_type = (Ewk_Mouse_Button_Type)0;
+  switch (button) {
+    case 1:
+      button_type = EWK_Mouse_Button_Left;
+      break;
+    case 2:
+      button_type = EWK_Mouse_Button_Middle;
+      break;
+    case 3:
+      button_type = EWK_Mouse_Button_Right;
+      break;
+  }
+
+  int px = x + left_;
+  int py = y + top_;
+  if (type == 0) {  // down event
+    button_type_ |= button_type;
```

Comment:
To do masking, the enum value of button_type must be in the format [1<<0, 1<<1, 1<<2](https://github.com/flutter-tizen/embedder/blob/master/flutter/shell/platform/embedder/embedder.h#L1197), not 1, 2, 3.
It's hard to confirm exactly because our device uses the right click as the back button, but I expect it will probably not work properly.